resources/js/Components/Listboxes/ProjectGroupListbox.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import {Listbox, ListboxButton, ListboxOption, ListboxOptions} from "@headlessui/vue";
+
+import {ChevronDownIcon} from "@heroicons/vue/outline";
+import Button from "@/Jetstream/Button.vue";
+import {CheckIcon} from "@heroicons/vue/solid";
+import {ref} from "vue";
+
+const props = defineProps({
```

Comment:
COULD: Variablen Definition können in JS "verkettet" werden. Damit sparst du dir das const keyword ständig.

const props = defineProps({}),
          currentGroup = ...,
         emit = defineEmits();